src/cljs/athens/views/node_page.cljs

```diff
@@ -505,17 +505,11 @@
          [:h1 (use-style title-style
                          {:data-uid uid
                           :class    "page-header"
-                          :on-click (fn [e] (navigate-uid uid e))})
+                          :on-click (fn [e]
+                                      (.. e preventDefault)
+                                      (dispatch [:editing/uid uid]))})
```

Comment:
So I agree we want to start editing the textarea when we click on the h1. However, I'm pretty sure `:editing/uid` is not actually used in any way for node pages. Not big deal either way.

More importantly, this change no longer lets the user navigate to a date page when they are on Daily Notes. Perhaps the `:on-click` handler should have an `if` statement for Daily Notes?

Reply:
Fixed the issue with Daily Notes.

Sorry, I just need to clarify. When you say `:editing/uid` is not used for the node page, why does the node page subscribe to it?

Reply:
I wasn't sure when I first created node page whether or not it needed `:editing/uid`. Arguably, it could use it.

The purpose of `:editing/uid` is to let the component know whether it is focused or not. This is more useful for blocks, where the caret could be moved to another block. Enter, backspace, up and down arrows are all places that could update :editing/uid.

In the case of block-page and node-page, a user might go to a child block if they press enter on the header (I believe this might've actually been your first PR!).

Arguably, pressing backspace on the 0th child block should take you user's focus back to the title page. Roam doesn't do this, but Notion does do this.

However, it's not super obvious node-page and block-page actually need this anymore. Maybe that would be code to remove in another PR.

Reply:
So far, what I found is that `:editing/uid` is used to check whether to apply `is-editing` css class. I might try to investigate more later.

Thanks for the feedback.